src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -54,9 +51,6 @@ public HelpCommand() {
 		this.avatar = ConfigManager.languageConfig.get().getString("Commands.Help.Elements.Avatar");
 		this.invalidTopic = ConfigManager.languageConfig.get().getString("Commands.Help.InvalidTopic");
 		this.usage = ConfigManager.languageConfig.get().getString("Commands.Help.Usage");
-		this.rpgUsage = ConfigManager.languageConfig.get().getString("Command.Help.RPGUsage");
```

Comment:
The NPE was most likely happening because this config path `Command.Help.RPGUsage` has a typo in it and should be `Commands.Help.RPGUsage` as shown on https://github.com/ProjectKorra/ProjectKorra/pull/1032/files#diff-b8a8bb8eeca244e5a83b003a6a0940acL169. However, we should strive to decouple Core from the side plugins so this change is okay for now. I think a better hot fix would just be fixing the config path.